### PR TITLE
Gradle 4.4, updated support version, build tools version, kotlin version, rxjava,…

### PR DESCRIPTION
**Pull request by tristanrichard**
… rxandroid …

### CODE REVIEW

**Comment by dlew** [comment]
The description says Gradle 4.1, but I don't see a Gradle update... (and I'd pick something more recent than 4.1 at this point).

That said, it fails CI because you also have to update the .travis.yml file.

**Comment by trr-amsiq** [comment]
Sorry I meant 4.6
I’ve updated the .travis.yml file

**Comment by tristanrichard** [comment]
@dlew

**Comment by dlew** [comment]
It's still not passing CI, not sure what you want me to do here. :P

**Comment by tristanrichard** [comment]
I don't know what to. do about the error message

**Comment by bmc08gt** [comment]
The SDK license needs to be accepted on the travis server

File: build.gradle [hunk 1]
```diff
@@ -27,18 +27,18 @@ ext {
     minSdkVersion = 14
     compileSdkVersion = 26
```
Comment (piotrek1543) [inline build.gradle]: @trr-amsiq @tristanrichard guys, you forgot to update `compileSdkVersion` to `27`, that's why Travis build still fails.

File: build.gradle [hunk 2]
```diff
@@ -27,18 +27,18 @@ ext {
...
     targetSdkVersion = compileSdkVersion
-    buildToolsVersion = '26.0.2'
+    buildToolsVersion = '27.0.3'
 
-    supportLibVersion = '26.0.2'
+    supportLibVersion = '27.1.0'
     lifecycleVersion = '1.0.0'
```
Comment (piotrek1543) [inline build.gradle]: For now, the newest version is `1.1.1`

**Comment by dlew** [comment]
Sorry, this PR kind of got lost in the busyness that was my life for the past few months. I'm closing this because now that I'm working on RxLifecycle 2 again (for a bit), half of these are out of date.

**Pull request closed without merging**